net-misc/sipp/sipp-3.6.1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2 ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +pcap sctp +ssl"
+
+DEPEND="sys-libs/ncurses:=
+	gsl? ( sci-libs/gsl:= )
+	pcap? (
+		net-libs/libpcap
+		net-libs/libnet:1.1
+	)
+	ssl? ( dev-libs/openssl:= )
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+
+	# version.h is in ${S}, and needs to be in ${S}/include for cmake to work.
+	ln -s ../version.h "${S}/include/" || die "Error creating version.h symlink"
+
+	cmake_src_prepare
+}
+
+src_configure() {
+	local u
+	local mycmakeargs=()
+
+	for u in ${IUSE//[+-]/}; do
+		mycmakeargs+=(-DUSE_${u^^}=$(usex ${u} 1 0))
```

Comment:
I think this hurts readability and is going to make it a pain if we have to adjust any of them. Could we lay them out explicitly?

Reply:
Sure ... I get why this seems nasty :).